src/components/Heading/components/Identity/Identity.tsx

```diff
@@ -173,30 +162,39 @@ export const Identity = () => {
         handleChange,
         handleBlur,
         handleSubmit
-      }: FormikProps<PayloadType>) => (
+      }) => (
         <form onSubmit={handleSubmit} className={`${styles.identity} identity`}>
-          {fields.map((field: FieldType) => (
-            <div key={field.name} className={styles.field}>
-              <label htmlFor={field.name}>{field.label}</label>
-              <div className='input-group'>
-                <input
-                  type='text'
-                  name={field.name}
-                  value={values[field.name]}
-                  autoComplete='off'
-                  onChange={handleChange}
-                  onBlur={handleBlur}
-                  className={classNames(styles.input, {
-                    [styles.invalid]: errors[field.name] && touched[field.name]
-                  })}
-                />
-
-                {errors[field.name] && touched[field.name] && (
-                  <span className={styles.error}>{errors[field.name]}</span>
-                )}
+          {fields.map((field: FieldType) => {
+            const currentValues = values as PayloadType;
```

Comment:
No need for cast. The type is inferred. Probably you didn't set the initialValues type correctly

Reply:
Fixed again. 👍